Poppy/erm-usage.json

```diff
@@ -0,0 +1,22 @@
+{
+  "id": "erm-usage-0.0.1",
+  "name": "erm-usage",
+  "version": "0.0.1",
+  "description": "ERM Usage platform installation for Poppy-SP1",
+  "platform": "base",
+  "dependencies": [],
+  "modules": [
+    {
+      "id": "mod-erm-usage-4.6.0",
+      "name": "mod-erm-usage",
+      "version": "4.6.0",
+      "url": "https://mdr-folio-eis-us-east-1-dev.s3.amazonaws.com/eureka/mod-erm-usage-4.6.0"
```

Comment:
```suggestion
      "url": "https://mdr-folio-eis-us-east-1-dev.s3.amazonaws.com/mod-erm-usage-4.6.0"
```

Reply:
please delete eureka/ from all mdr urls. It's no custom MDs for such modules. Use the standard one